### ISSUE TITLE
haskell-language-server for GHC 9.0.x compiled without hls-tactics-plugin

### ISSUE THREAD

**Issue by R-VdP** [NixOS/nixpkgs#177759]
### Describe the bug
`hls-tactics-plugin` is missing from `haskell-language-server-wrapper --list-plugins`

### Steps To Reproduce
1. `nix-shell -p haskellPackages.haskell-language-server`
2. `haskell-language-server-wrapper --list-plugins`
3. Observer the absence of `hsl-tactics-plugin`

### Expected behavior
The plugin should be present.

### Screenshots
```
$ haskell-language-server-wrapper --list-plugins
No 'hie.yaml' found. Try to discover the project type!
Run entered for haskell-language-server-wrapper(haskell-language-server-wrapper) Version 1.7.0.0 x86_64 ghc-9.0.2
Current directory: /home/ramses
Operating system: linux
Arguments: ["--list-plugins"]
Cradle directory: /home/ramses
Cradle type: Default

Tool versions found on the $PATH
cabal:          Not found
stack:          2.7.5
ghc:            9.0.2


Consulting the cradle to get project GHC version...
Project GHC version: 9.0.2
haskell-language-server exe candidates: ["haskell-language-server-9.0.2","haskell-language-server"]
Launching haskell-language-server exe at:/run/current-system/sw/bin/haskell-language-server-9.0.2
2022-06-15T13:42:57.468506Z | Info | No log file specified; using stderr.
LSPRecorderCallback
alternateNumberFormat
callHierarchy
changeTypeSignature
class
eval
floskell
fourmolu
ghcide-code-actions-bindings
ghcide-code-actions-fill-holes
ghcide-code-actions-imports-exports
ghcide-code-actions-type-signatures
ghcide-completions
ghcide-core
ghcide-hover-and-symbols
ghcide-type-lenses
haddockComments
hlint
importLens
moduleName
ormolu
pragmas
qualifyImportedNames
refineImports
rename
retrie
selectionRange
splice
```

### Metadata
```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.0, NixOS, 22.11 (Raccoon), 22.11pre384739.90cd5459a1f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.9.1`
 - channels(ramses): `""`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by maralorn** [comment]
Weird I thought I enabled it but apparently something is more intricate than I thought.

**Comment by maralorn** [comment]
e

**Comment by R-VdP** [comment]
@maralorn did you get a chance to look into this further?

**Comment by barendventer** [comment]
So Wingman is not supported on GHC later than version 8.10. Pointing VSCode at `haskell-language-server-wrapper` as its executable and using it in a project configured to use GHC 8.10.7, I get this output from the nix-shell:

```
[nix-shell:~/source/repos/jazzlump]$ haskell-language-server-wrapper --list-plugins
No 'hie.yaml' found. Try to discover the project type!
Run entered for haskell-language-server-wrapper(haskell-language-server-wrapper) Version 1.7.0.0 x86_64 ghc-9.0.2
Current directory: /home/kaidelong/source/repos/jazzlump
Operating system: linux
Arguments: ["--list-plugins"]
Cradle directory: /home/kaidelong/source/repos/jazzlump
Cradle type: Cabal

Tool versions found on the $PATH
cabal:          3.6.2.0
stack:          Not found
ghc:            8.10.7


Consulting the cradle to get project GHC version...
Project GHC version: 8.10.7
haskell-language-server exe candidates: ["haskell-language-server-8.10.7","haskell-language-server"]
Launching haskell-language-server exe at:/nix/store/r08msgdb3rka4xpdh6klvc9kmx5fwb69-haskell-language-server-1.7.0.0/bin/haskell-language-server-8.10.7
2022-09-18T05:25:07.627159Z | Info | No log file specified; using stderr.
LSPRecorderCallback
alternateNumberFormat
callHierarchy
changeTypeSignature
. #etc etc
.
.
tactics
```

So I think the package is actually behaving correctly. Hopefully the problem will resolve itself once wingman is supported in later GHC versions?

**Comment by maralorn** [comment]
Yeah, while working on the migration to hls 1.8 I understood the flagsystem better. tactics support is simply disabled for ghc 9.0 in hls 1.7 here:

https://github.com/haskell/haskell-language-server/blob/b5a37f7fc360596899cb2945f363030f44156415/haskell-language-server.cabal#L267

This will be fixed in a few days when hls 1.8 lands in nixpkgs.
